woocommerce/woocommerce-android
# Add unit tests to functionalities related to Shipping Labels M4: Add/create package

To fix #4464

This PR adds unit tests to various functionalities added in the add/create package screen.

### To test:

Run unit tests, check especially for these files and make sure they all succeed:
- ShippingLabelCreatePackageViewModelTest.kt
- ShippingLabelCreateCustomPackageViewModelTest.kt
- ShippingLabelCreateServicePackageViewModelTest.kt

Also please check each added test's name and please let me know if anything's unclear and could be better named.

Update release notes:
